test/bity_test.dart

```diff
@@ -162,16 +173,22 @@ void main() {
       server.enqueue(httpCode: 400, body: cannedResponse);
 
       expect(
-          () => client.createCryptoToFiatOrder(
-                inputCurrency: inputCurrency,
-                outputAmount: outputAmount,
-                outputCurrency: outputCurrency,
-                outputIban: iban,
-                owner: Owner("", "", "", "", ""),
-                reference: reference,
-              ),
-          throwsA(TypeMatcher<InvalidBankAddress>()
-              .having((e) => e.toString(), "toString()", contains(''))));
+          await client.createCryptoToFiatOrder(
+            inputCurrency: inputCurrency,
+            outputAmount: outputAmount,
+            outputCurrency: outputCurrency,
+            outputIban: iban,
+            owner: Owner("", "", "", "", ""),
+            reference: reference,
+          ),
+          equals({
+            'errors': [
+              {
+                'code': 'invalid_bank_address',
+                'message': 'The bank account owner address is invalid.'
+              }
+            ]
+          }));
```

Comment:
I think the three tests above can be unified into one since it's only forwarding of the msg.